doc/vision/vision.md

```diff
@@ -0,0 +1,147 @@
+We give a research area, for example we enter the name "Web Databases" or "Join Optimisation".
+Our Research Paper Graph will display all papers from this area and we can filter and sort these papers by 
+their time, distance, and relevance.
+
+Our Research Paper Analyser considers which paper families contain the most papers that have our research area as a buzzword. 
+These families represent our research area. 
+
+![../img/img_02.png](../img/img_02.png)
+
+### Researcher Families
+Given is researcher Bob and we ask ourselves with which researchers he has frequent exchange. 
+For this, we pass our Research Paper Analyser the researcher's name -- Bob -- and it presents Bob in a graph of
+researchers that he communicates with.
+
+For this, our Research Paper Analyser observes which paper families Bob is involved in and 
+considers other researchers from these families. 
+
+## Architecture
+Our Research Paper Analyzer is composed of a scraper, a database, an analyzing backend and a graph-heavy frontend:
+- The scraper stores the meta information and references of research papers in a database. 
+It reads this data from a research API such as SemanticScholar, Clarivate, and ArXiv and it 
+adapts the format of the data to the schema of the underlying database. 
+- The research graphs in the frontend represent firstly families of related research papers, 
+secondly research areas with their respective work, 
+and thirdly families of communicating researchers. 
+The user can filter and sort the graphs according to his research interests. 
+- The Research Paper Analyser acts as a backend. 
+It translates the user's actions into equivalent database queries and 
+passes the resulting data to the frontend in an efficiently usable format. 
+
+![../img/img_03.png](../img/img_03.png)
+
+## Technologies
+- As our database we use PostgreSQL.
+- We implement our scraper in Python and let it access the SemanticScholar Paper API.
+- We build our backend on Django using Python.
+- We write our frontend in Typescript and React, integrating frameworks for graph visualization.
+
+### PostgreSQL
+PostgreSQL is a free, open source object-relational database management system. 
+It supports SQL language, extensible data types, operators, functions and aggregates as well as the ACID conditions.
+Reliability, robustness, and performance characterize PostgreSQL.
+Link: [https://www.postgresql.org](https://www.postgresql.org)
+
+### Python
+Python is a universal, interpreted, and higher programming language. 
+Because it is both concise and readable, we can easily learn and understand Python. 
+Data is particularly well analysed with Python.
+Link: [https://www.python.org](https://www.python.org) 
+
+### SemanticScholar Paper API
+SemanticScholar stores metainformation about papers, their authors, and their references. 
+We can download this information and access it through an API.
+Link: [https://api.semanticscholar.org](https://api.semanticscholar.org) 
+
+### Django
```

Comment:
I looked around a little because I never did a backend in python before. While there are two popular web frameworks (Django and Flask) I think Flask might be a little easier to use for us. So maybe we should use Flask instead of Django? Sorry for the premature pointer towards Django :see_no_evil: